404 on Peer @iamstarkov/listr-update-renderer@0.4.1

### Description
When doing a install of this module, I get a 404 on one of the peers

https://github.com/SamVerschueren/listr-update-renderer . 

On NPM it says `npm i @iamstarkov/listr-update-renderer`
but in the readme it states `npm install --save listr-update-renderer` 

<!--
If this is a feature request, explain why it should be added. Specific
use-cases are best.

If this is a bug, please check that it can be reproduced in the latest version
of `lint-staged`. Furthermore, please provide as much *relevant* info as
possible. Including what is the expected and actual behavior.
-->

### Steps to reproduce

<!-- Tell us how to reproduce this issue. -->
`npm i lint-staged`
### Debug Logs

<!--
Run `lint-staged` in debug mode using the command line flag `--debug`. You can
either modify your pre-commit script/command to include the flag or invoke
`lint-staged` manually(`./node_modules/.bin/lint-staged --debug`). Read more
about debug mode here -
https://github.com/okonet/lint-staged#command-line-flags.

NOTE: This requires `lint-staged` version >= `6.0.0`.
-->
<details>

<summary>expand to view</summary>

```
npm ERR! code E404
npm ERR! 404 Not Found: @iamstarkov/listr-update-renderer@latest
```

</details>

### Environment

<!-- Tell us about your development environment -->
- **OS:** macOS <!-- ex: Windows 10, Ubuntu 16.04, macOS High Sierra -->
- **Node.js:** 10.14.1<!-- ex: v8.9.1 -->
- **`lint-staged`:** 8.1.0 <!-- ex: v6.0.0 -->

--- a/package.json
+++ b/package.json
@@ -26,7 +26,6 @@
     }
   },
   "dependencies": {
-    "@iamstarkov/listr-update-renderer": "0.4.1",
     "chalk": "^2.3.1",
     "commander": "^2.14.1",
     "cosmiconfig": "^5.0.2",
@@ -39,6 +38,7 @@
     "is-glob": "^4.0.0",
     "is-windows": "^1.0.2",
     "listr": "^0.14.2",
+    "listr-update-renderer": "^0.5.0",
     "lodash": "^4.17.11",
     "log-symbols": "^2.2.0",
     "micromatch": "^3.1.8",

